### PR TITLE
Drop UnblindedElement output from test vectors.

### PR DESCRIPTION
**Pull request by chris-wood**
I missed this in #227. Thanks, @bytemare!

### CODE REVIEW

**Comment by chris-wood** [comment]
@alxdavids can you please review and merge if this work?

File: poc/oprf.sage [hunk 1]
```diff
@@ -205,6 +206,25 @@ class VerifiableClientContext(ClientContext,Verifiable):
...
         return unblinded_elements
 
+    def finalize_batch(self, xs, blinds, evaluated_elements, blinded_elements, proof):
+        assert(len(blinds) == len(evaluated_elements))
+        assert(len(evaluated_elements) == len(blinded_elements))
+
+        unblinded_elements = self.unblind_batch(blinds, evaluated_elements, blinded_elements, proof)
+
+        outputs = []
+        finalizeDST = _as_bytes("VOPRF06-Finalize-") + self.context_string
```
Comment (alxdavids) [inline poc/oprf.sage:216]: Nit: should we use a different DST when batching?

**Review by alxdavids — APPROVED** [review]
LGTM, one nit around batching, but I don't have a strong opinion either way.

File: poc/oprf.sage (hunk 1 above)
Comment (chris-wood) [reply poc/oprf.sage:216]: Since this is just an optimization, I think the output should be the same.

**Comment by chris-wood** [comment]
I [made these changes in CIRCL](https://github.com/cloudflare/circl/pull/217) and they seem to pass. So I'm merging this as-is.

**Pull request merged**